modules/ROOT/partials/configuration/valid_children.adoc

```diff
@@ -7,9 +7,9 @@ The valid_children enables you to control what child elements can exists within
 
 The syntax for this option is a comma separated list of parents with elements that should be added/removed as valid children for that element. So for example `'+body[style]'` would add style as a valid child of body.
```

Comment:
A line above contains a ref that needs to include the full ref as it refers to something in another file:
Line 6: ```xref:schema[`schema`]```